app/javascript/listings/components/AllListings.jsx

```diff
@@ -13,7 +13,7 @@ const AllListings = ({
   loadNextPage,
 }) => {
   return (
-    <main class="crayons-layout__content">
+    <main class="crayons-layout__content" id="articles-list">
```

Comment:
refactor (blocking): The `id` fixes in conjunction with converting `<div />` to `<main />` tags here and all below enable skip links on all the pages where they were not present. The only thing is the `id` is `articles-list` which makes sense on the main page, but for other pages it does not.

We could probably go with a more generic `id` value like `skip-link`, or something to that effect as it makes it clear what it represents for any page the skip link is on.

Reply:
I did `articles-list` because I wasn't sure on how you wanted to have it re: naming conventions. I can go back in and make the changes if you want to go with `skip-link` which makes more sense for the inside pages.

Reply:
I think `main-content` would make a lot of sense!